csrankings-a.csv

```diff
@@ -2437,8 +2437,9 @@ Azer Bestavros,Boston University,http://www.bu.edu/cs/azer-bestavros,S5B0QmMAAAA
 Aziz Mohaisen,University of Central Florida,http://cs.ucf.edu/~mohaisen,q387AywAAAAJ
 Aziz Salah,Université du Québec à Montréal,https://www.labunix.uqam.ca/~salah_a/3ardh.php,NOSCHOLARPAGE
 Aznul Qalid Md Sabri,University of Malaya,https://umexpert.um.edu.my/aznulqalid,oX5-84kAAAAJ
+Azza Abouzied,NYU Abu Dhabi,https://nyuad.nyu.edu/en/academics/divisions/science/faculty/azza-abouzied.html,RkddXygAAAAJ
 Azzam Ibrahim,University of Jordan,http://eacademic.ju.edu.jo/azzam.sleit,yZ42G9YAAAAJ
-Azzam Mourad,Lebanese American University,http://sas.lau.edu.lb/csm/people/azzam-mourad.php,p3DTnQQAAAAJ
+Azzam Mourad,NYU Abu Dhabi,https://nyuad.nyu.edu/en/academics/divisions/science/faculty/azzam-mourad.html,p3DTnQQAAAAJ

```

Comment:
Not found here: https://nyuad.nyu.edu/en/academics/faculty.html?_charset_=UTF-8&division=science&program=computer-science-major&affiliation=NYU+Abu+Dhabi&keywords=

Reply:
(Home page is dead)